modules/passkey/src/types/solc.d.ts

```diff
@@ -0,0 +1,41 @@
+declare module 'solc' {
+  export interface CompilerInput {
```

Comment:
Fairly sure these types aren't used anywhere - and I think are inaccurate (in particular `CompilerOutput` specifically depends on the `outputSelection` of the input). I would remove them if they aren't used.

Reply:
They're used in the codesize task; without type declaration for `solc`, the project won't compile. They might be inaccurate since I generated them using the perplexity AI tool - however, I believe they're still safer than just declaring everything as any, like [here](https://github.com/safe-global/safe-modules/blob/main/modules/4337/types/solc.d.ts). I'll take one more look and fix the inaccuracies

Reply:
Ahh, now I see what you mean, you're right